src/components/Editor/Alarm/AlarmListItem.vue

```diff
@@ -267,7 +267,7 @@ export default {
 			return true
 		},
 		canChangeAlarmType() {
-		  return this.forceEventAlarmType !== null && this.alarm.type !== this.forceEventAlarmType
+		  return !this.forceEventAlarmType && this.alarm.type !== this.forceEventAlarmType
```

Comment:
Well, I don't know what was really intended here to be honest. I just know that the logic is definitely broken without this fix, since no selection at all is possible anymore.
From my perspective, both can make sense.

Reply:
From https://github.com/nextcloud/calendar/pull/3997#issue-1150306573
> If a reminder doesn't match the enforced type (event created before enforcement or another client) then the user decide to stay on the old value or switch to the enforced one

For me, this makes sense. Otherwise, the user could be stuck forever with a reminder type that is forbidden by the admin.

Reply:
@max65482 Ah, thanks for the pointer. I will adjust it then.

Reply:
I fixed it. If the alarm type is `EMAIL`, but the forced type is `DISPLAY` one can change it to `DISPLAY` now (and vice-versa). The usability is a bit weird still, because the radio boxes vanish as soon as one clicks the forced type, but this should be handled in a different PR if desired.